lib/src/workmanager.dart

```diff
@@ -182,7 +192,7 @@ class JsonMapperHelper {
     assert(uniqueName != null);
     assert(taskName != null);
     return {
-      "isInDebugMode": _isInDebugMode,
+      "isInDebugMode": isInDebugMode ?? false,
```

Comment:
Does this make sense?
When can this be `null`?

Reply:
This was in response to a comment you had earlier about making it a named argument with a default value

Reply:
But yeah `registerPeriodicTask` and `toRegisterMethodArgument` are only used internally so we control the call site. But it doesn't hurt to add default value anyway, right?

Reply:
I rather reason about non nullability when I have to ...

Can this value ever be null or not? 
If it will always be non null that don't add this default case here, it should be handled in the `initialize`.